NeoML/include/NeoML/TraditionalML/Svm.h

```diff
@@ -23,12 +23,14 @@ limitations under the License.
 
 namespace NeoML {
 
+class IThreadPool;
+
 DECLARE_NEOML_MODEL_NAME( SvmBinaryModelName, "FmlSvmBinaryModel" )
 
 // Support-vector machine binary classifier
 class NEOML_API ISvmBinaryModel : public IModel {
 public:
-	~ISvmBinaryModel() override;
+	~ISvmBinaryModel() override {}
```

Comment:
The code of the destructor was moved on purpose. Otherwise we may encounter some problems because of weak linking on GCC (when `typeinfo( *svmBinaryModel )` returns different result in external code and in internal code)

Reply:
Mb I'm a bit overreacting (but this problem is 100% relevant for simple structs, see `CLstmDesc` for example) but better save than sorry)

Reply:
`~ISvmBinaryModel() override = default;`

Reply:
fixed